dp-import-cantabular-dataset.nomad

```diff
@@ -0,0 +1,140 @@
+job "dp-import-cantabular-dataset" {
+  datacenters = ["eu-west-1"]
+  region      = "eu"
+  type        = "service"
+
+  update {
+    stagger          = "60s"
+    min_healthy_time = "30s"
+    healthy_deadline = "2m"
+    max_parallel     = 1
+    auto_revert      = true
+  }
+
+  group "web" {
```

Comment:
This shouldn't have the `web` group in it. This service only runs in publishing. You'll have to redo what was done in #3